ui/app/src/components/CRUDButton/CRUDButton.tsx

```diff
@@ -50,6 +51,30 @@ export function CRUDButton(props: CRUDButtonProps) {
     );
   }
 
+  if (!hasPermission && action && scope && project) {
+    const errorMessage =
+      project === GlobalProject
+        ? `Missing '${action}' global permission for '${scope}' kind`
+        : `Missing '${action}' permission in '${project}' project for '${scope}' kind`;
+
+    return (
+      <Tooltip title={errorMessage} placement="top">
+        <span>
+          <Button
+            variant={variant}
+            color={color}
+            size="small"
+            sx={{ textTransform: 'uppercase' }}
+            onClick={onClick}
+            disabled={true}
```

Comment:
```suggestion
disabled
```
should work without giving the boolean value.